compose.yml

```diff
@@ -8,7 +8,7 @@ services:
       diesel setup &&
       diesel migration generate --diff-schema create_match_data &&
       diesel migration run;
-      cargo watch -x run"
+      cargo watch -x test"
```

Comment:
Let's stick to test here for now, but will need to start thinking about how to set up different images for dev and prod

Reply:
noted, will read more about this